docs/features/security/auth/jwt.md

```diff
@@ -62,4 +62,55 @@ public class Startup
         });
     }
 }
+```
+
+### Custom Claim validation
+
+It allows validating not only on the audience claim in the JWT token, but any type of custom claim that needs to be verified
+
+```csharp
```

Comment:
This looks outdated based on the code, no? Ideally you don't have to pass the claims to the reader which is the case I think?

Reply:
And additionally, this should be in the preview docs, right 🤔 . Not sure how this slipped the first review 😅 .

Reply:
Ah, and this additional section is purely about the claims, the section above talks about options where you don't have to pass the claim check. So, it's good, I guess?

Reply:
I don't think so because based on the extension code you don't pass the claims to the reader but the extension itself?

Reply:
Aha! Now, I see it.

Reply:
I'll also update the extensions so you're not required to pass along the configure options function every time and only can pass the claim checks (+ update the docs of course).